docs/basic/getting-started/forward-create-ref.md

```diff
@@ -54,8 +54,10 @@ export const FancyButton = React.forwardRef((
 
 If you are grabbing the props of a component that forwards refs, use [`ComponentPropsWithRef`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a05cc538a42243c632f054e42eab483ebf1560ab/types/react/index.d.ts#L770).
 
-More info: https://medium.com/@martin_hotell/react-refs-with-typescript-a32d56c4d315
-
+More info: 
```

Comment:
I think it would be better to add here more than just a link.

Can you add here a section of **Generic Forward Refs**?

Reply:
Typing Generic forwardRefs

```js
type Props = { children: React.ReactNode; type: "submit" | "button" };
export type Ref = HTMLButtonElement;
export const FancyButton = React.forwardRef<Ref, Props>((props, ref) => (
  <button ref={ref} className="MyClassName" type={props.type}>
    {props.children}
  </button>
));
```